apps/accounts/api/automations/gather_account.py

```diff
@@ -76,6 +76,7 @@ class GatheredAccountViewSet(OrgBulkModelViewSet):
     model = GatheredAccount
     search_fields = ("username",)
     filterset_class = GatheredAccountFilterSet
+    extra_filter_backends = [NodeFilterBackend]
     ordering = ("status",)
     serializer_classes = {
         "default": serializers.DiscoverAccountSerializer,
```

Comment:
Based on the given code snippet, there do not appear to be any major issues with regard to coding standards or logic that would indicate it needs significant modifications. The code is clear and appears well-commented; however, you should consider the following points:

1. Ensure consistent indentation (4 spaces per line) throughout the file.

2. It's recommended to use `@property` when fetching related fields instead of hard-coding them directly into view functions which can lead to less predictable behavior if more complex models are added later on.

3. There might be an issue with your Django version compatibility, as some parts of the code could fail due to older versions. Consider upgrading Django to at least 3.x.